docs/dpdk/mlnx-dpdk/mlnx-pod_testpmd_connectx4.yaml

```diff
@@ -8,20 +8,20 @@ spec:
   containers:
   - name: testpmd
     image: <DPDK testpmd image>
-    imagePullPolicy: Never
+    imagePullPolicy: IfNotPresent
     securityContext:
      capabilities:
-        add: ["IPC_LOCK"]
+        add: ["IPC_LOCK", "NET_RAW"]
```

Comment:
Why is `NET_RAW` added?

Reply:
Mellanox dpdk driver is beforcated so we need additional capabilities from the kernel see https://github.com/intel/sriov-network-device-plugin/issues/123

Reply:
With crio, `NET_RAW` is one of the default capabilities passed to each container/pod. User doesn't need to explicitly specify it in pod spec.

Reply:
With crio, `NET_RAW` is one of the default capabilities assigned to each container/pod. User doesn't need to explicitly specify it in pod spec.

Reply:
but with docker we need this. do you want us to add comment that is not required for crio?

Reply:
From this doc, it looks like docker keeps `net_raw` as default capability: https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities
Is the doc outdated?

Reply:
I see I remember that it wasn't but I see I am mistaken. @Mmduh-483 please remove the NET_RAW as it is the default.